src/components/NavBar/NavBar.jsx

```diff
@@ -32,6 +31,7 @@ const NavBar = () => {
             <i className="fa fa-caret-down">User</i>
           </button>
           <div className='dropdown-content'>
+          <Link to="SearchUser">Search</Link>
```

Comment:
Same as comment above - I'd consider having a link to take you to view all the users rather than a dropdown.